src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -262,7 +271,28 @@ public String getType() {
         public Map<String, Object> getCustomStates() {
             return customStates;
         }
-
+        
+        public List<Object> getFormat() {
+        	HttpFuzzerReflectionDetectorStateHighlighter value = new HttpFuzzerReflectionDetectorStateHighlighter();

```

Comment:
Were you suggesting to move the whole decoding process of customState itself to the HttpFuzzResultsContentPanel class (or HttpFuzzerResultsTableModel class) or simply just this line?